### ISSUE TITLE
RTX HDR: Subtly lower peak brightness on minimal GUI or fullscreen

### ISSUE THREAD

**Issue by 11xx** [Aleksoid1978/VideoRenderer#199]
The RTX HDR support works great when there's at least one GUI element on the screen and on windowed mode. But when using minimal mode (1) i.e. no GUI elements but the video from the MPC-BE window, the peak brightness effect of RTX HDR gets slightly lower for some reason. Same thing happens on fullscreen but this time the progress bar on the bottom doesn't make the peak brightness "peak" correctly.

Here's an example:

https://github.com/user-attachments/assets/b18ca9d8-f9ef-4dc5-ae16-e70bd6ae250a

The camera exagerated the effect a bit, the irl difference would be the equivalent of like 20-40 nits. (This is a hdr 400 screen)

(As a workaround I would enable the little status bar at the bottom and make it black and use the player maximized.)

**Comment by v0lt** [comment]
There are three modes:
1) window (you see the desktop or other windows);
2) window maximized to full screen;
3) exclusive full screen (when you can't bring up the context menu).

Perhaps the driver is too smart and tries to handle all these cases. Or maybe its logic depends on the output frame size.
Try unchecking "Use for resizing".

**Comment by 11xx** [comment]
Unchecked, but same thing. I also disabled it on NCP. I've looked at #146, doesn't seem to be the same issue.

However, after you mentioned it, I realized I didn't test between cases 2 and 1 where just the *maximized* window on minimal mode is on the screen, *with the taskbar hidden*, and it happens with that too.

Then the weird part that includes somewhat of a easy workaround I guess:
- Some actions on the second monitor like triggering some window animations would make the brightness flash peak brightness but then go back after the animation ended, while;
- Either dragging a window on the second monitor "close enough" to the main monitor (like 50 pixels from showing up on the screen with the video), or snapping a Window to the side closer to the main monitor also "fixes" it, and leaving it there (on the second monitor), even behind other windows retains "fixed" the effect.
- On the second monitor, if there's no window "close enough" to the main one, it will dim regardless.
- Finally, tested with a single display setup, same issue.

I'm nowhere familiar with MPC ~~and this may sound dumb~~ but could something like a single semi-transparent GUI pixel be placed in a corner to trick the driver in these cases?

Well, in my case the second monitor will be on anyway so this helped already, thanks :)
